src/sentry/models/projectownership.py

```diff
@@ -287,6 +288,19 @@ def handle_auto_assignment(cls, project_id, event):
                 }
             )
 
+            activity = Activity.objects.filter(
+                group=event.group, type=ActivityType.ASSIGNED.value
+            ).order_by("-datetime")
+            if activity:
+                auto_assigned = activity[0].data.get("integration")
+                if not auto_assigned:
+                    analytics.record(
+                        "autoassignment.post_manual_assignment",
+                        organization_id=event.group.project.organization_id,
+                        project_id=project_id,
+                        group_id=event.group.id,
+                    )
+
```

Comment:
Thinking about just returning early here now - initially I thought we'd gather metrics first and then stop auto-assigning when it's already been manually assigned, but we can still gather metrics for when we _would_ have done it.

Reply:
I think I'll just merge this for now and go back tomorrow to return early and write a test 😎